vscode/src/commands/services/CommandCodeLenses.ts

```diff
@@ -147,11 +169,15 @@ export class EditorCodeLenses implements vscode.CodeLensProvider {
     }
 }
 
-const editorCodeLenses = {
+const commandLenses = {
     cody: {
         title: '$(cody-logo) Cody',
         command: 'cody.editor.codelens.click',
         tooltip: 'Open command menu',
     },
-    inline: { title: 'Inline Chat', command: 'cody.editor.codelens.click', tooltip: 'Ask Cody inline' },
+    test: {
+        title: '$(cody-logo) Add Tests',
+        command: 'cody.editor.codelens.click',
+        tooltip: 'Generate new test cases for the selected test suit',
```

Comment:
```suggestion
        tooltip: 'Generate new test cases',
```

I'd personally keep this simple IMO, users can already see it's attached to the suite